server/router.js

```diff
@@ -235,17 +235,26 @@ router.post('/api/annotation', (req, res) => {
 /*
 Get annotations of an article
 Input:
-  req.body.uri: URI of article
+  req.query.uri: URI of article
 Output: Returns json file of the article's annotations or error.
 */
 router.get('/api/article/annotations', (req, res) => {
   let user = null;
-  const topLevelOnly = req.query.topLevelOnly;
+  let pagination_options = {};
+  if (req.query.page && req.query.size) {
+    pagination_options.skip = req.query.size * req.query.page;
+    pagination_options.limit = req.query.size * 1;
+  }
+  if (req.query.sort) { // TODO: work on this
```

Comment:
ok so I haven't totally figured out how to include sorting on querystring, so right now this is irrelevant, but leaving so as to remind myself that sorting also needs to happen